packages/frontend/src/components/tags.tsx

```diff
@@ -2,21 +2,28 @@ import { Sport } from "@esp-group-one/types";
 
 interface Tagging {
   sportName: Sport;
+  active: boolean;
+  onClick: () => void;
 }
 
-export function Tag({ sportName }: Tagging) {
+export function Tag({ sportName, active, onClick }: Tagging) {
```

Comment:
I don't like the addition of `onClick` (and especially making it a requirement). I think the parent should have an `onClick` which then contains the tag